src/components/expandable-text/expandable-text.tsx

```diff
@@ -25,7 +25,7 @@ interface ExpandableTextProps {
 
 const ReadMoreLessButton = ({isTextExpanded, readLessLabel, readMoreLabel, onClick, onKeyDown, ...otherProps}) => {
   return (
-    <div tabIndex={0} className={styles.moreButtonText} onClick={onClick} onKeyDown={onKeyDown} {...otherProps}>
+    <div role="button" tabIndex={0} className={styles.moreButtonText} onClick={onClick} onKeyDown={onKeyDown} {...otherProps}>
```

Comment:
seems pretty straightforward.
my only request would be to check whether this changes the screen reader behavior for the more button which appears for audio entry description:
https://github.com/kaltura/playkit-js-ui/blob/92c6ddcecdd6b4f3765c8bbb8012a694d8fcaaef/src/components/audio-entry-details/audio-entry-details.tsx#L115

Reply:
it add the word "button" to the announcement and this is expected cause it's indeed a button and need it from WCAG requirements.